lesson-two/bootstrap-starter.html

```diff
@@ -11,11 +11,24 @@
     <!-- Boostrap's optional theme-->
     <link rel="stylesheet" href="bootstrap/css/bootstrap-theme.min.css"/>
 
+  <style>
+    .services {width: 100%;}
+  </style>
+
   </head>
   <body>
+    <h1>Nathan Cook</h1>
+    <!--<nav></nav>-->
     <div class="container">
-      <div class="col-xs-12 col-sm-6">a</div>
-      <div class="col-xs-12 col-sm-6">b</div>
+      <div class="col-xs-12 col-sm-12">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
```

Comment:
The two classes on this div are superfluous. You only need `col-xs-12` to set it full width all the time.